### PR TITLE
Start kafka in travis

### PR DESCRIPTION
**Pull request by boy-uber**
Start zookeeper/kafka cluster in .travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,6 +47,13 @@ before_install:
   - pip install --user ccm
   - pip install --user cqlsh==5.0.3
   - sudo apt-get install gcc g++ -y
+  - wget http://www.us.apache.org/dist/kafka/0.10.0.1/kafka_2.11-0.10.0.1.tgz -O kafka.tgz
+  - mkdir -p kafka && tar xzf kafka.tgz -C kafka --strip-components 1
+  - nohup bash -c "cd kafka && bin/zookeeper-server-start.sh config/zookeeper.properties &"
+  - nohup bash -c "cd kafka && bin/kafka-server-start.sh config/server.properties &"
+  - sleep 10
```
Comment (boy-uber) [inline .travis.yml]: I see. Let check whether there is a better way :)

Comment (thuningxu) [inline .travis.yml]: There is no reason to sleep here. The build process takes many minutes. By the time the build finishes kafka should be up already.

File: .travis.yml [hunk 2]
```diff
@@ -47,6 +47,13 @@ before_install:
...
+  - kafka/bin/kafka-topics.sh --create --partitions 2 --replication-factor 1 --topic test_topic_1 --zookeeper localhost:2181
+  - kafka/bin/kafka-topics.sh --zookeeper localhost:2181 --list
```
Comment (thuningxu) [inline .travis.yml]: Not sure why we need to do it here. Move to ##script## section (after ##make##)?

File: .travis.yml (hunk 1 above)
Comment (boy-uber) [inline .travis.yml]: Sleep here is to make sure the following topic creation could succeed (after kafka server starts and runs).

File: .travis.yml (hunk 2 above)
Comment (boy-uber) [inline .travis.yml]: I decided to totally remove the topic creation, since the kafka cluster will auto create topic.

**Comment by boy-uber** [comment]
I updated the code and removed sleep there, also remove the topic creation/list script since topic will be auto created.

**Comment by boy-uber** [comment]
Merge to master: start kafka cluster in travis CI

**Pull request merged**